Dockerfile

```diff
@@ -45,10 +45,13 @@ ENV DEBIAN_FRONTEND noninteractive
 # between older and newer distros (in this case it was /etc/rpc
 # existing only in the old launcher.gcr.io image) and using dpgk -S
 # <file> to determine which package supplies it, can be helpful.
-RUN apt-get update && apt-get dist-upgrade -y && apt-get install -y --no-install-recommends \
+RUN apt-get update && apt-get dist-upgrade -y && apt-mark unhold libcap2 && apt-get install --reinstall -y --no-install-recommends \
```

Comment:
For future reference, we might add a comment about why we need to unhold libcap2 and add the `--reinstall` flag.

Reply:
Done